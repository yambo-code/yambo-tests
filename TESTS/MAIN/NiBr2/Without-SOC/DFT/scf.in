&control
              prefix = 'NiBr2'
          pseudo_dir = '../../pseudo'
           verbosity = 'high'
          wf_collect = .true.
/&end
&system
           celldm(1) = 6.9451076104
           celldm(3) = 3.5996562475955844
             degauss = 0.005
             ecutwfc = 70
               ibrav = 4
            lspinorb = .false.
                 nat = 3
            noncolin = .false.
               nspin = 2
                ntyp = 2
         occupations = 'smearing'
starting_magnetization(1) = 1
starting_magnetization(2) = 0
/&end
&electrons
            conv_thr = 1e-08
/&end


ATOMIC_SPECIES
  Ni  58.6934               Ni.upf
  Br   79.904               Br.upf
ATOMIC_POSITIONS { crystal }
 Ni   0.0000000000   0.0000000000   0.0000000000
 Br   0.6666666700   0.3333333300   0.1038888250
 Br   0.3333333300   0.6666666700  -0.1038888250
K_POINTS { automatic }
  3  3  1  0  0  0
