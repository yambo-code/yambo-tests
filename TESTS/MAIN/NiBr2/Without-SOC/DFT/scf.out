
     Program PWSCF v.6.5 starts on 17May2021 at 16:52:21 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     MPI processes distributed on     1 nodes
     Waiting for input...
     Reading input from standard input

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         931     931    265                82711    82711   12671



     bravais-lattice index     =            4
     lattice parameter (alat)  =       6.9451  a.u.
     unit-cell volume          =    1044.3080 (a.u.)^3
     number of atoms/cell      =            3
     number of atomic types    =            2
     number of electrons       =        32.00
     number of Kohn-Sham states=           20
     kinetic-energy cutoff     =      70.0000  Ry
     charge density cutoff     =     280.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=   6.945108  celldm(2)=   0.000000  celldm(3)=   3.599656
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (  -0.500000   0.866025   0.000000 )  
               a(3) = (   0.000000   0.000000   3.599656 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.577350 -0.000000 )  
               b(2) = (  0.000000  1.154701  0.000000 )  
               b(3) = (  0.000000 -0.000000  0.277804 )  


     PseudoPot. # 1 for Ni read from file:
     ../../pseudo/Ni.upf
     MD5 check sum: 14a7e289c5fff9cacaba393040e62f02
     Pseudo is Norm-conserving + core correction, Zval = 18.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1372 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     PseudoPot. # 2 for Br read from file:
     ../../pseudo/Br.upf
     MD5 check sum: 41f672ce76ec6f6480a05aabe46aac99
     Pseudo is Norm-conserving + core correction, Zval =  7.0
     Generated using ONCVPSP code by D. R. Hamann
     Using radial grid of 1110 points,  6 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
                l(5) =   2
                l(6) =   2

     atomic species   valence    mass     pseudopotential
        Ni            18.00    58.69340     Ni( 1.00)
        Br             7.00    79.90400     Br( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Ni           1.000
        Br           0.000

     12 Sym. Ops., with inversion, found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  2     180 deg rotation - cart. axis [1,0,0]        

 cryst.   s( 2) = (     1          0          0      )
                  (    -1         -1          0      )
                  (     0          0         -1      )

 cart.    s( 2) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym =  3     120 deg rotation - cryst. axis [0,0,1]       

 cryst.   s( 3) = (     0          1          0      )
                  (    -1         -1          0      )
                  (     0          0          1      )

 cart.    s( 3) = ( -0.5000000 -0.8660254  0.0000000 )
                  (  0.8660254 -0.5000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  4     120 deg rotation - cryst. axis [0,0,-1]      

 cryst.   s( 4) = (    -1         -1          0      )
                  (     1          0          0      )
                  (     0          0          1      )

 cart.    s( 4) = ( -0.5000000  0.8660254  0.0000000 )
                  ( -0.8660254 -0.5000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  5     180 deg rotation - cryst. axis [0,1,0]       

 cryst.   s( 5) = (    -1         -1          0      )
                  (     0          1          0      )
                  (     0          0         -1      )

 cart.    s( 5) = ( -0.5000000 -0.8660254  0.0000000 )
                  ( -0.8660254  0.5000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym =  6     180 deg rotation - cryst. axis [1,1,0]       

 cryst.   s( 6) = (     0          1          0      )
                  (     1          0          0      )
                  (     0          0         -1      )

 cart.    s( 6) = ( -0.5000000  0.8660254  0.0000000 )
                  (  0.8660254  0.5000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym =  7     inversion                                    

 cryst.   s( 7) = (    -1          0          0      )
                  (     0         -1          0      )
                  (     0          0         -1      )

 cart.    s( 7) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000 -1.0000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym =  8     inv. 180 deg rotation - cart. axis [1,0,0]   

 cryst.   s( 8) = (    -1          0          0      )
                  (     1          1          0      )
                  (     0          0          1      )

 cart.    s( 8) = ( -1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym =  9     inv. 120 deg rotation - cryst. axis [0,0,1]  

 cryst.   s( 9) = (     0         -1          0      )
                  (     1          1          0      )
                  (     0          0         -1      )

 cart.    s( 9) = (  0.5000000  0.8660254  0.0000000 )
                  ( -0.8660254  0.5000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym = 10     inv. 120 deg rotation - cryst. axis [0,0,-1] 

 cryst.   s(10) = (     1          1          0      )
                  (    -1          0          0      )
                  (     0          0         -1      )

 cart.    s(10) = (  0.5000000 -0.8660254  0.0000000 )
                  (  0.8660254  0.5000000  0.0000000 )
                  (  0.0000000  0.0000000 -1.0000000 )


      isym = 11     inv. 180 deg rotation - cryst. axis [0,1,0]  

 cryst.   s(11) = (     1          1          0      )
                  (     0         -1          0      )
                  (     0          0          1      )

 cart.    s(11) = (  0.5000000  0.8660254  0.0000000 )
                  (  0.8660254 -0.5000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


      isym = 12     inv. 180 deg rotation - cryst. axis [1,1,0]  

 cryst.   s(12) = (     0         -1          0      )
                  (    -1          0          0      )
                  (     0          0          1      )

 cart.    s(12) = (  0.5000000 -0.8660254  0.0000000 )
                  ( -0.8660254 -0.5000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


     point group D_3d (-3m) 
     there are  6 classes
     the character table:

       E     2C3   3C2'  i     2S6   3s_d 
A_1g   1.00  1.00  1.00  1.00  1.00  1.00
A_2g   1.00  1.00 -1.00  1.00  1.00 -1.00
E_g    2.00 -1.00  0.00  2.00 -1.00  0.00
A_1u   1.00  1.00  1.00 -1.00 -1.00 -1.00
A_2u   1.00  1.00 -1.00 -1.00 -1.00  1.00
E_u    2.00 -1.00  0.00 -2.00  1.00  0.00

     the symmetry operations in each class and the name of the first element:

     E        1
          identity                                               
     2C3      3    4
          120 deg rotation - cryst. axis [0,0,1]                 
     3C2'     2    5    6
          180 deg rotation - cart. axis [1,0,0]                  
     i        7
          inversion                                              
     2S6      9   10
          inv. 120 deg rotation - cryst. axis [0,0,1]            
     3s_d     8   11   12
          inv. 180 deg rotation - cart. axis [1,0,0]             

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Ni  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           Br  tau(   2) = (   0.5000000   0.2886751   0.3739641  )
         3           Br  tau(   3) = (  -0.0000000   0.5773503  -0.3739641  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Ni  tau(   1) = (  0.0000000  0.0000000  0.0000000  )
         2           Br  tau(   2) = (  0.6666667  0.3333333  0.1038888  )
         3           Br  tau(   3) = (  0.3333333  0.6666667 -0.1038888  )

     number of k points=     3  Gaussian smearing, width (Ry)=  0.0050
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.1111111
        k(    2) = (   0.0000000   0.3849002   0.0000000), wk =   0.6666667
        k(    3) = (   0.3333333   0.5773503   0.0000000), wk =   0.2222222

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.1111111
        k(    2) = (   0.0000000   0.3333333   0.0000000), wk =   0.6666667
        k(    3) = (   0.3333333   0.3333333   0.0000000), wk =   0.2222222

     Dense  grid:    82711 G-vectors     FFT dimensions: (  40,  40, 135)

     Dynamical RAM for                 wfc:       3.15 MB

     Dynamical RAM for     wfc (w. buffer):      22.06 MB

     Dynamical RAM for           str. fact:       2.52 MB

     Dynamical RAM for           local pot:       0.00 MB

     Dynamical RAM for          nlocal pot:       8.51 MB

     Dynamical RAM for                qrad:       2.97 MB

     Dynamical RAM for          rho,v,vnew:      17.46 MB

     Dynamical RAM for               rhoin:       5.82 MB

     Dynamical RAM for            rho*nmix:      40.39 MB

     Dynamical RAM for           G-vectors:       5.36 MB

     Dynamical RAM for          h,s,v(r/c):       0.29 MB

     Dynamical RAM for          <psi|beta>:       0.02 MB

     Dynamical RAM for                 psi:      12.61 MB

     Dynamical RAM for                hpsi:      12.61 MB

     Dynamical RAM for      wfcinit/wfcrot:       6.31 MB

     Estimated static dynamical RAM per process >     118.77 MB

     Estimated max dynamical RAM per process >     144.29 MB
     Generating pointlists ...
     new r_m :   0.2838 (alat units)  1.9707 (a.u.) for type    1
     new r_m :   0.2838 (alat units)  1.9707 (a.u.) for type    2

     Initial potential from superposition of free atoms

     starting charge   31.99973, renormalised to   32.00000
     Starting wfcs are   18 randomized atomic wfcs +    2 random wfcs

     total cpu time spent up to now is        1.7 secs

     Self-consistent Calculation

     iteration #  1     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  6.7

     total cpu time spent up to now is        5.9 secs

     total energy              =    -393.08807451 Ry
     Harris-Foulkes estimate   =    -393.26123826 Ry
     estimated scf accuracy    <      10.19716813 Ry

     total magnetization       =     5.78 Bohr mag/cell
     absolute magnetization    =     5.85 Bohr mag/cell

     iteration #  2     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  4.5

     total cpu time spent up to now is       10.9 secs

     total energy              =    -395.77876594 Ry
     Harris-Foulkes estimate   =    -405.35628753 Ry
     estimated scf accuracy    <      30.82184243 Ry

     total magnetization       =     0.95 Bohr mag/cell
     absolute magnetization    =     1.00 Bohr mag/cell

     iteration #  3     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.8

     total cpu time spent up to now is       14.3 secs

     total energy              =    -397.51190068 Ry
     Harris-Foulkes estimate   =    -398.38298410 Ry
     estimated scf accuracy    <       4.35224628 Ry

     total magnetization       =     1.99 Bohr mag/cell
     absolute magnetization    =     2.03 Bohr mag/cell

     iteration #  4     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.2

     total cpu time spent up to now is       17.1 secs

     total energy              =    -398.28563543 Ry
     Harris-Foulkes estimate   =    -399.02354486 Ry
     estimated scf accuracy    <       3.16748875 Ry

     total magnetization       =     2.22 Bohr mag/cell
     absolute magnetization    =     2.38 Bohr mag/cell

     iteration #  5     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  9.90E-03,  avg # of iterations =  2.5

     total cpu time spent up to now is       19.9 secs

     total energy              =    -398.35924582 Ry
     Harris-Foulkes estimate   =    -398.57267806 Ry
     estimated scf accuracy    <       0.71881629 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.07 Bohr mag/cell

     iteration #  6     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  2.25E-03,  avg # of iterations =  2.5

     total cpu time spent up to now is       22.9 secs

     total energy              =    -398.47970288 Ry
     Harris-Foulkes estimate   =    -398.48795955 Ry
     estimated scf accuracy    <       0.03306766 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.05 Bohr mag/cell

     iteration #  7     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.03E-04,  avg # of iterations =  3.7

     total cpu time spent up to now is       26.0 secs

     total energy              =    -398.48433981 Ry
     Harris-Foulkes estimate   =    -398.48449010 Ry
     estimated scf accuracy    <       0.00138824 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.07 Bohr mag/cell

     iteration #  8     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.34E-06,  avg # of iterations =  4.2

     total cpu time spent up to now is       29.8 secs

     total energy              =    -398.48463473 Ry
     Harris-Foulkes estimate   =    -398.48546626 Ry
     estimated scf accuracy    <       0.00314057 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.08 Bohr mag/cell

     iteration #  9     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.34E-06,  avg # of iterations =  2.0

     total cpu time spent up to now is       32.4 secs

     total energy              =    -398.48495904 Ry
     Harris-Foulkes estimate   =    -398.48497175 Ry
     estimated scf accuracy    <       0.00004305 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.08 Bohr mag/cell

     iteration # 10     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  1.35E-07,  avg # of iterations =  3.2

     total cpu time spent up to now is       35.5 secs

     total energy              =    -398.48496875 Ry
     Harris-Foulkes estimate   =    -398.48496912 Ry
     estimated scf accuracy    <       0.00000107 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.08 Bohr mag/cell

     iteration # 11     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.34E-09,  avg # of iterations =  2.3

     total cpu time spent up to now is       38.6 secs

     total energy              =    -398.48496894 Ry
     Harris-Foulkes estimate   =    -398.48496927 Ry
     estimated scf accuracy    <       0.00000124 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.08 Bohr mag/cell

     iteration # 12     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  3.34E-09,  avg # of iterations =  2.0

     total cpu time spent up to now is       41.2 secs

     total energy              =    -398.48496909 Ry
     Harris-Foulkes estimate   =    -398.48496909 Ry
     estimated scf accuracy    <       0.00000001 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.08 Bohr mag/cell

     iteration # 13     ecut=    70.00 Ry     beta= 0.70
     Davidson diagonalization with overlap
     ethr =  4.04E-11,  avg # of iterations =  2.7

     Magnetic moment per site:
     atom:    1    charge:   16.5239    magn:    1.3815    constr:    0.0000
     atom:    2    charge:    4.7881    magn:    0.2328    constr:    0.0000
     atom:    3    charge:    4.7881    magn:    0.2328    constr:    0.0000

     total cpu time spent up to now is       44.3 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000 ( 10351 PWs)   bands (ev):

  -104.6494 -65.0154 -65.0154 -64.9735 -16.1468 -15.3109  -5.4275  -4.5655
    -4.5655  -2.6056  -2.6056  -2.4135  -2.1307  -1.8779  -1.8779  -0.6239
    -0.6239   3.4570   4.7263   5.1312

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000

          k = 0.0000 0.3849 0.0000 ( 10350 PWs)   bands (ev):

  -104.6495 -65.0177 -65.0153 -64.9747 -15.5719 -15.1260  -5.5184  -5.0562
    -4.4630  -4.2321  -3.1827  -2.9356  -2.5426  -2.1525  -2.0164  -1.2178
    -0.8075   4.2226   5.4999   6.2758

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000

          k = 0.3333 0.5774 0.0000 ( 10281 PWs)   bands (ev):

  -104.6490 -65.0149 -65.0149 -64.9732 -15.1342 -15.1342  -5.4970  -5.0351
    -5.0351  -4.0630  -4.0630  -3.8499  -2.3696  -2.0816  -2.0816  -1.2750
    -1.2750   4.8779   7.1707   7.1707

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   0.0000   0.0000   0.0000

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000 ( 10351 PWs)   bands (ev):

  -103.1497 -63.3815 -63.3815 -63.3611 -15.8881 -15.0259  -5.1868  -3.3609
    -3.3609  -2.1503  -1.6049  -1.6049  -1.5847  -1.5847  -1.3165   0.8068
     0.8068   3.6050   4.8017   5.2330

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   0.0000
     0.0000   0.0000   0.0000   0.0000

          k = 0.0000 0.3849 0.0000 ( 10350 PWs)   bands (ev):

  -103.1498 -63.3840 -63.3814 -63.3623 -15.2982 -14.8289  -5.3034  -4.6689
    -3.6304  -3.2665  -2.7775  -2.3935  -1.5516  -1.3028  -1.1853   0.3792
     0.6885   4.4437   5.6214   6.3811

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   0.0000
     0.0000   0.0000   0.0000   0.0000

          k = 0.3333 0.5774 0.0000 ( 10281 PWs)   bands (ev):

  -103.1494 -63.3809 -63.3809 -63.3607 -14.8406 -14.8406  -5.3154  -4.7020
    -4.7020  -3.6336  -3.0045  -3.0045  -1.5040  -1.2455  -1.2455   0.4131
     0.4131   5.1398   7.3124   7.3124

     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   0.0000
     0.0000   0.0000   0.0000   0.0000

     the Fermi energy is    -0.0046 ev

!    total energy              =    -398.48496910 Ry
     Harris-Foulkes estimate   =    -398.48496910 Ry
     estimated scf accuracy    <          8.0E-10 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -947.87699703 Ry
     hartree contribution      =     477.20730543 Ry
     xc contribution           =     -63.57724801 Ry
     ewald contribution        =     135.76197051 Ry
     smearing contrib. (-TS)   =      -0.00000000 Ry

     total magnetization       =     2.00 Bohr mag/cell
     absolute magnetization    =     2.08 Bohr mag/cell

     convergence has been achieved in  13 iterations

     Writing output data file ./NiBr2.save/

     init_run     :      1.49s CPU      1.53s WALL (       1 calls)
     electrons    :     42.49s CPU     42.63s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.92s CPU      0.93s WALL (       1 calls)
     wfcinit:atom :      0.01s CPU      0.02s WALL (       6 calls)
     wfcinit:wfcr :      0.88s CPU      0.88s WALL (       6 calls)
     potinit      :      0.30s CPU      0.33s WALL (       1 calls)
     hinit0       :      0.23s CPU      0.23s WALL (       1 calls)

     Called by electrons:
     c_bands      :     35.61s CPU     35.70s WALL (      13 calls)
     sum_band     :      4.10s CPU      4.11s WALL (      13 calls)
     v_of_rho     :      2.53s CPU      2.58s WALL (      14 calls)
     v_h          :      0.07s CPU      0.07s WALL (      14 calls)
     v_xc         :      2.47s CPU      2.52s WALL (      14 calls)
     mix_rho      :      0.36s CPU      0.38s WALL (      13 calls)

     Called by c_bands:
     init_us_2    :      0.49s CPU      0.50s WALL (     162 calls)
     cegterg      :     34.33s CPU     34.42s WALL (      78 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :     26.78s CPU     26.83s WALL (     331 calls)
     g_psi        :      0.09s CPU      0.09s WALL (     247 calls)
     cdiaghg      :      0.22s CPU      0.22s WALL (     325 calls)
     cegterg:over :      3.31s CPU      3.32s WALL (     247 calls)
     cegterg:upda :      2.31s CPU      2.33s WALL (     247 calls)
     cegterg:last :      1.40s CPU      1.40s WALL (      81 calls)

     Called by h_psi:
     h_psi:calbec :      3.77s CPU      3.78s WALL (     331 calls)
     vloc_psi     :     19.28s CPU     19.32s WALL (     331 calls)
     add_vuspsi   :      3.64s CPU      3.64s WALL (     331 calls)

     General routines
     calbec       :      3.77s CPU      3.78s WALL (     331 calls)
     fft          :      1.01s CPU      1.01s WALL (     264 calls)
     ffts         :      0.10s CPU      0.10s WALL (      26 calls)
     fftw         :     20.40s CPU     20.44s WALL (   10136 calls)

     Parallel routines

     PWSCF        :     44.15s CPU     44.35s WALL


   This run was terminated on:  16:53: 6  17May2021            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
